### PR TITLE
CADENZA-37830 CADENZA-37929 feat: Document support to show, fetch and download reports generated from report template embedding targets as PDF

### PR DESCRIPTION
**Pull request by sebastianbarth-disy**
UI Change reviewed by Jürgen Wäldele

### CODE REVIEW

**Review by jkissel — CHANGES_REQUESTED** [review]

File: sandbox.html [hunk 1]
```diff
@@ -440,8 +440,8 @@
   </div>
   <div>
     <label>
-      <input type="checkbox" name="jasperReportAsPdf" id="jasperReportAsPdf"/>
-      Show Jasper Report as PDF
+      <input type="checkbox" name="requestAsPdf" id="requestAsPdf"/>
+      PDF (Jasper Report, Report Template)
```
[the remark this answers is not in the document]
Comment (sebastianbarth-disy) [reply sandbox.html:444]: Report Templates / Berichtsvorlagen is a defined term. But I see your point: In traditional Cadenza speak we would "Generate and export Reports from Report Templates". 

Report Templates can be "Exported" via Manage Workbook dialog, which will open the "Exporting Report" dialog. In the Embeddable Content dialog they are referenced as "Report".

I'll pass this to PO/TR/UX for discussion.

File: src/cadenza.js [hunk 2]
```diff
@@ -281,7 +281,7 @@ export class CadenzaClient {
    * @param {PageSource | EmbeddingTargetId} source - The source to show
    * @param {object} [options]
    * @param {DataType} [options.dataType] - Set to 'pdf' for views of type "JasperReports report"
-   *     to show the report PDF directly, without any Cadenza headers or footers.
+   *     to show the report PDF directly, without any Cadenza headers or footers and embedding targets for report templates.
```
[the remark this answers is not in the document]
Comment (sebastianbarth-disy) [reply src/cadenza.js:284]: Will improve...

File: src/cadenza.js [hunk 3]
```diff
@@ -953,7 +953,7 @@ export class CadenzaClient {
    * @param {EmbeddingTargetId} source - The workbook view to fetch data from.
    * @param {DataType} dataType - The data type you want to get back from the server.
    *   Currently, `"csv"`, `"excel"` and `"json"` are supported for table and indicator views
-   *   and `"pdf"` for views of type "JasperReports report".
+   *   and `"pdf"` for views of type "JasperReports report" and embedding targets for report templates.
```
[the remark this answers is not in the document]
Comment (sebastianbarth-disy) [reply src/cadenza.js:956]: Adhoc Reports are not part of this. Curious, what Workbook Reports are compared to Reports generated from Report Templates. PO/UX/TR will sort out...

File: CHANGELOG.md [hunk 4]
```diff
@@ -5,6 +5,8 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 This project uses a version scheme based on the Cadenza main version in the format x.x.y, where x.x is the Cadenza main version and y a functional change or bugfix.
 
 ## Unreleased
+### Added
+- Support to show, fetch and download reports generated from report template embedding targets as PDF
```
[the remark this answers is not in the document]
Comment (sebastianbarth-disy) [reply CHANGELOG.md:9]: I thought I group this change together with the corresponding change of Cadenza to support it. But ok, I will limit it to Cadenza JS.

File: sandbox.html (hunk 1 above)
[the remark this answers is not in the document]
Comment (jkissel) [reply sandbox.html:444]: :+1:

**Review by jkissel — APPROVED** [review]

**Pull request merged**